zerver/migrations/0429_fix_audit_log_objects_for_group_based_stream_settings.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.1.6 on 2023-02-17 12:06
+import json
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def fix_audit_log_objects_for_group_based_stream_settings(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
```

Comment:
Let's add a docstring here containing the content in the commit message -- that this adds the `property_name` field to any `STREAM_GROUP_BASED_SETTING_CHANGED` audit log entries that were created before the previous commit.